### ISSUE TITLE
Problems activating template

### ISSUE THREAD

**Issue by rafaelpoliveira** [itmicus/zabbix#18]
I did everything that was said in readme but I'm stuck in add lines to zabbix.conf. Where I can find that configuration file. I'm new in Zabbix World!!!

I tried to put that lines in Zabbix agent configuration file, on Hyper-V Server - Failed!
I tried to put that lines in /etc/httpd/conf.d/zabbix.conf - Failed!

Where I have to put that lines?

**Comment by itmicus** [comment]
Hello!
stop stop stop man, you must edit only config on zabbix client, not on a zabbix server.
Please, look at readme for Template carefully 

**Create 2 folders in zabbix agent** folder: \scripts\
\zabbix_agentd\
and copy the files
hyperv_host.ps1 to \scripts\
hyperv_host.conf to \zabbix_agentd\

**Add lines to zabbix.conf**

Include=C:\Program Files\zabbix-agent\zabbix_agentd\*.conf
UnsafeUserParameters=1
Timeout=10

Restart Zabbix Agent

**Comment by rafaelpoliveira** [comment]
Ok…

So, I have to change that Include path based on my path, correct?

I’ve tried in the zabbix client but did not work! Let me try again and I will ;et you know about that.

Regards,

Rafael

On 13 Aug 2018, at 03:39, itmicus <notifications@github.com<mailto:notifications@github.com>> wrote:


Hello!
stop stop stop man, you must edit only config on zabbix client, not on a zabbix server.
Please, look at readme for Template carefully

Create 2 folders in zabbix agent folder: \scripts
\zabbix_agentd
and copy the files
hyperv_host.ps1 to \scripts
hyperv_host.conf to \zabbix_agentd\

Add lines to zabbix.conf

Include=C:\Program Files\zabbix-agent\zabbix_agentd*.conf
UnsafeUserParameters=1
Timeout=10

Restart Zabbix Agent

**Comment by pankajverma90** [comment]
Zabbix agent fails to start after adding the lines to zabbix_agentd.conf file:

Include=C:\Program Files\Zabbix Agent\zabbix_agentd\*.conf
UnsafeUserParameters=1
Timeout=10

It says " Windows could not start the Zabbix Agent service on Local Computer. Error 1053: The service did not respond to the start or control request in a timely fashion. "
I have put the exact path where config file has been kept. 

PS: Agent works fine if the line with Include parameter is removed or commented.

**Comment by wmorgue** [comment]
Hi @rafaelpoliveira 
Issue is still relevant?
